test/functional/vimscript/has_spec.lua

```diff
@@ -68,4 +68,10 @@ describe('has()', function()
       eq(0, funcs.has('wsl'))
     end
   end)
+
+  it('does not change v:shell_error', function()
+    funcs.system('bash -c "exit 73"')
```

Comment:
these tests run on windows. a platform-agnostic way to force an exit code is to run `nvim` with `:cquit`:

```lua
local nvim_prog = helpers.nvim_prog

funcs.system({nvim_prog, '+73cquit'})
```

Reply:
Ah. Thanks.